src/charm.py

```diff
@@ -203,10 +203,13 @@ def _create_env(self):
         Returns:
             env: dictionary of environment variables
         """
-        if "superset-secret-key" in self.config:
+        if self.config["superset-secret-key"]:
```

Comment:
What changed that it is guaranteed to have that key now? Was the previous version wrong?

Reply:
It was correct until the last PR in which I introduced the [structured config](https://github.com/canonical/superset-k8s-operator/pull/19). With the structured config this field became guaranteed. I should have updated it as part of this previous PR.